_episodes/01-package-setup.md

```diff
@@ -205,11 +207,14 @@ $ cd molecool
 ### The `__init__.py` file
 
 The `__init__.py` file is a special file recognized by the Python interpreter which makes a directory into a package. This file can be blank in some cases, however, we will use it to define how the user interacts with the functions in our package.
+*TODO: Cite section on defining the interface, where we can also mention `__all__` and `_` prefixed names.*
```

Comment:
This may be more appropriate in the section ["Deciding Package Structure"](https://education.molssi.org/python-package-best-practices/05-package-structure/index.html).

Reply:
Maybe what he means is adding hyperlink to __init__.py in Deciding Package Structure?

Reply:
Yes, cross-linking with the later more thorough discussions would be good. The terseness of the description here seems appropriate, but
* it shouldn't imply that this section completely describes how we define the public interface, and
* references to public interface in Deciding Package Structure (or elsewhere) should include conventions on import behavior, documentation visibility, and linter semantics w.r.t. `__all__` and underscore-prefixed names, at least in passing.